src/bot.js

```diff
@@ -167,6 +167,43 @@ client.on('messageCreate', async (message) => {
 
 					}
 
+					else if (message.content.includes(config.command_end)) {
+						// collect tags and add close tag
+						let initialTags = [closeTag[0].id,...postTags];
+						let tags = [...new Set(initialTags)];
+
+						// send embed message upon executing the close command
+						await message.channel.send({ 
+							embeds: [
+								sendEmbedMessage(`${config.reminder_close}`)
+							],
+							content: `🔔`
```

Comment:
No longer needed. Let's remove the `content`

Reply:
@warengonzaga we can safely remove it right?

Reply:
https://github.com/thirdweb-dev/support-discord-bot/pull/68/commits/1c7c24a20af24e4f1fa24ed95fd1a577cea6b889

Reply:
yes, it is safe to remove.